airbyte-integrations/connectors/source-airtable/source_airtable/schema_helpers.py

```diff
@@ -89,17 +91,22 @@ def get_json_schema(table: Dict[str, Any]) -> Dict[str, str]:
             name: str = SchemaHelpers.clean_name(field.get("name"))
             original_type: str = field.get("type")
             options: Dict = field.get("options", {})
-            exec_type: str = options.get("result", {}).get("type") if options else None
-            field_type: str = exec_type if exec_type else original_type
+            options_result: Dict = options.get("result", {})
```

Comment:
do we need an `if options else None` here?

Reply:
Kept that to ensure `options: null` case is covered, just in case)